### ISSUE TITLE
headless

### ISSUE THREAD

**Issue by trollhus** [fugasjunior/arma-server-manager#81]
Hi, You have made it supereasy to create a server ! Thank you! :)

But as there is no builtin support for headless yet is it possible for me to run a manual ./armaserver_x64 -client within the container or is there a better way to do it?

**Comment by fugasjunior** [comment]
Hi, thanks for the feedback!

As you say, until the support for HC is built into the app, you can manually run the client as you suggest.

An alternative approach which I personally use is to create a script that starts the headless client and set it up as an additional server. You can follow the steps to do so here: https://github.com/fugasjunior/arma-server-manager#additional-servers.

Sample script `a3hc.sh`:
```bash
#!/bin/bash
cd /home/steam/armaservermanager/servers/ARMA3 && ./arma3server_x64 -client -connect=127.0.0.1:2302 -password=abcd -mod=@CBA_A3 -mod=@ace ...
```

You just need to make sure the script is accessible in the container. Let me know if you need any help setting this up.

**Comment by trollhus** [comment]
Thank you. That shell script works perfectly! :) Do you use screen or something if you want to use more than one HCs? 
Also if I wish to edit bandwith, min error etc, how should I do it?  I can't find a file or where to designate my "network" file for startup parameters.  Its not a big issue tho as I mostly play on my own or with a few friends :)

Again, I hope this get more popular because its extremely easy to setup. You should share it along the forums. 
Great work!

Thank you!

**Comment by fugasjunior** [comment]
Perfect, glad it's working! If you need to run multiple HCs, as you say, you can use either `screen` or `tmux` to do that. Alternatively if you decide to use the "additional servers" function, you can create two distinct additional servers which run the same script, so then you could run the HCs using the UI.

Regarding the network config, currently there's no way to do that using the UI. However according to the documentation, Arma 3 generates a default config `Arma3.cfg` in the user profile directory if none is specified. So you can attach to the container and find the file (`find / -name Arma3.cfg`) and try editting it manually. That's an oversight on my mind and I'll likely add the option to edit these settings soon.

**Comment by fugasjunior** [comment]
Support for launching headless clients through UI is now available in release v1.2.0.
